spec/time_service_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'spec_helper'
+require_relative '../time_service.rb'
+
+# Create TimeService for testing purposes comprised of the interface we are testing.
+class TimeService < Hoodoo::Services::Service
+  comprised_of TimeInterface
+end
+
+describe 'Time Service' do
+  include Rack::Test::Methods
+
+  # Create rack builder object for the test.
+  context "get /1/Time/now" do
+    let(:app) {
+
+      Rack::Builder.new do
```

Comment:
Formatting - remove empty line
```suggestion
    let(:app) {
      Rack::Builder.new do
```